examples/comp.py

```diff
@@ -0,0 +1,56 @@
+import gt4py
```

Comment:
Be sure to delete this and any other extraneous files before merging (based on what you said in the PR description).